pkg/k8shandler/fluentd.go

```diff
@@ -361,6 +361,19 @@ func newFluentdPodSpec(cluster *logging.ClusterLogging, proxyConfig *configv1.Pr
 		{Name: "POD_IP", ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{APIVersion: "v1", FieldPath: "status.podIP"}}},
 	}
 
+	forwarderSpec := logging.ForwarderSpec{}
+	if cluster.Spec.Forwarder != nil {
+		forwarderSpec = *cluster.Spec.Forwarder
+		if forwarderSpec.Fluentd.Buffer != nil {
+			if forwarderSpec.Fluentd.Buffer.ChunkLimitSize != "" {
+				fluentdContainer.Env = append(fluentdContainer.Env, v1.EnvVar{Name: "BUFFER_SIZE_LIMIT", Value: string(forwarderSpec.Fluentd.Buffer.ChunkLimitSize)})
```

Comment:
I don't believe this is the correct solution.  These values should be calculated as part of the run.sh

Reply:
I thought in that way too then we should ssh into the fluentd pod and grep these values. Is this what you feel is more correct way?
BUFFER_SIZE_LIMIT=$(cat fluent.conf | grep chunk_limit_size | awk 'NR==1{print $2}' | sed -e "s/[Kk]/*1024/g;s/[Mm]/*1024*1024/g;s/[Gg]/*1024*1024*1024/g;s/i//g" | bc)

Reply:
Other than this we don't have access to values because it's not present in fluentd pod environment